CHANGELOG.md

```diff
@@ -155,8 +155,13 @@
 
 ## `net`
 
+### Version "0.6.0"
+
+- Update http crate to 1.0
+
 ### Version "0.5.0"
 
+- Update http crate to 1.0
```

Comment:
Could you please remove this line from v0.5.0 since it is moved to v0.6.0?
Could you please also add the pull request number like other change log lines?